PlanningPoker.Services/ISummaryRepository.cs

```diff
@@ -1,15 +1,22 @@
 namespace PlanningPoker.Services
 {
+    using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
-    using PlanningPoker.Shared;
+    using Shared;
 
     public interface ISummaryRepository
     {
         Task<SummaryDTO> CreateAsync(SummaryCreateUpdateDTO summary);
 
         Task<SummaryDTO> FindAsync(int summaryId);
```

Comment:
De andre her bør vel bare fjernes, da de ikke skal bruges.

Reply:
You are right, my young padawan.